gabbi/tests/test_utils.py

```diff
@@ -87,3 +93,42 @@ def test__colorize_missing_color(self):
 
         message = utils._colorize('BLUE', 'hello')
         self.assertNotEqual('hello', message)
+
+
+class CreateUrlTest(unittest.TestCase):
```

Comment:
Okay, but in my mind it is your way that is inconsistent: initial caps, _even if_ it is an acronym.